core/modules/map.rs

```diff
@@ -1210,11 +1210,37 @@ impl ModuleMap {
         status == v8::ModuleStatus::Evaluated
           || status == v8::ModuleStatus::Errored
       );
+
+      fn get_module_map(arg: v8::Local<v8::Value>) -> Rc<ModuleMap> {
+        let module_map = v8::Local::<v8::External>::try_from(arg).unwrap();
+        unsafe { &*(module_map.value() as *const Rc<ModuleMap>) }.clone()
+      }
+      fn wake_module(
+        _scope: &mut v8::HandleScope<'_>,
+        args: v8::FunctionCallbackArguments<'_>,
+        _rv: v8::ReturnValue,
+      ) {
+        let module_map = get_module_map(args.data());
+        module_map.module_waker.wake();
+      }
+
       let promise = v8::Local::<v8::Promise>::try_from(value)
         .expect("Expected to get promise as module evaluation result");
-      let empty_fn =
-        crate::runtime::bindings::create_empty_fn(tc_scope).unwrap();
-      promise.catch(tc_scope, empty_fn);
```

Comment:
Interesting! We've had this one for the longest time. Would be great if it's not longer needed